core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -377,6 +359,28 @@ static SequenceNumbersService sequenceNumberService(
             seqNoStats.getGlobalCheckpoint());
     }
 
+    private SeqNoStats loadSeqNoStats(EngineConfig.OpenMode openMode) throws IOException {
+        final SeqNoStats seqNoStats;
+        switch (openMode) {
+            case OPEN_INDEX_AND_TRANSLOG:
+                final long globalCheckpoint = Translog.readGlobalCheckpoint(engineConfig.getTranslogConfig().getTranslogPath());
+                seqNoStats = store.loadSeqNoStats(globalCheckpoint);
+                break;
```

Comment:
I'd just return `store.loadSeqNoStats()` here rather than `break`ing and `return`ing at the bottom. Not sure if that's against our style tho?

Reply:
Done